backend/src/core/services/job.service.ts

```diff
@@ -119,17 +119,6 @@ const sendCampaign = async ({
   return Promise.all(jobs)
 }
 
-/**
- * @see ../backend/src/database/migrations/20210501121338-create-stop-jobs-function.js
- * @param campaignId
- */
-const stopCampaign = (campaignId: number): Promise<any> | undefined => {
-  return Campaign.sequelize?.query('SELECT stop_jobs(:campaignId);', {
```

Comment:
should we also create a migration to kill this function in our DB?

Reply:
Let's keep it first for the record, we might want to use this one instead of deleting in the delete campaign endpoint. This can be refactored out if we want to later in the db refactor